src/Dialogs/StripeDialog.vala

```diff
@@ -427,5 +437,43 @@ public class AppCenter.Widgets.StripeDialog : Gtk.Dialog {
 
         return data.str;
     }
+
+    private string get_stripe_error (Json.Object error_object) {
```

Comment:
You don't check if there's `type` member in the `error_object` so, this could lead to some fatals.